scala-api/src/main/scala/com/cloudera/livy/ScalaJobContext.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cloudera.livy
+
+import java.io.File
+
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.SQLContext
+import org.apache.spark.sql.hive.HiveContext
+import org.apache.spark.streaming.StreamingContext
+
+class ScalaJobContext(context: JobContext) {
+
+  def sc: SparkContext = context.sc().sc
+
+  def sqlctx: SQLContext =  context.sqlctx()
+
+  def hivectx: HiveContext = context.hivectx()
+
+  def streamingctx: StreamingContext = context.streamingctx().ssc
+
+  def createStreamingContext(batchDuration: Long) = context.createStreamingContext(batchDuration)
+
+  def stopStreamingContext = context.stopStreamingCtx()
```

Comment:
`def stopStreamingContext()`; methods that do things should have `()` to indicate they're not simple getters.